src/client/entity-editor/common/linked-entity.tsx

```diff
@@ -91,7 +91,8 @@ class LinkedEntity extends React.Component<any, any> {
 				{nameComponent}
 				&nbsp;&nbsp;
 				{
-					disambiguation &&
+					author ? <span className="disambig"><small>({author})</small></span> :
+						disambiguation &&
```

Comment:
Same comment as above, we want to show the disambiguation.
I'm not 100% certain we want to show the author in this component.

I wonder how that will look like considering one of the main relationships we show for works is "author wrote work".
This is what I envision which doesn't add much:
![image](https://user-images.githubusercontent.com/6179856/152796709-773b6090-bb27-49f3-ab4a-7ffa6ab339a4.png)